daemon/container_unix.go

```diff
@@ -640,7 +640,7 @@ func (container *Container) updateNetworkSettings(n libnetwork.Network, ep libne
 		return err
 	}
 
-	if container.hostConfig.NetworkMode == runconfig.NetworkMode("bridge") {
+	if container.hostConfig.NetworkMode == runconfig.NetworkMode("default") {
 		networkSettings.Bridge = container.daemon.config.Bridge.Iface
```

Comment:
@mavenugo The purpose of this patch is to expose the bridge name to runc so that when restoring the container, criu can attach the veth to the bridge. Since I do not see libnetwork/bridge exposes the default bridge name to the docker daemon, I have to use networkSettings.Bridge, which seems not being assigned any value.